map2loop/stratigraphic_column.py

```diff
@@ -40,9 +40,15 @@ def __init__(self):
                 ("maxAge", float),
                 ("group", str),
                 ("supergroup", str),
-                ("ThicknessMean", float),
-                ("ThicknessMedian", float),
-                ("ThicknessStdDev", float),
+                ("ThicknessMean_ThicknessCalculatorAlpha", float),
+                ("ThicknessMedian_ThicknessCalculatorAlpha", float),
+                ("ThicknessStdDev_ThicknessCalculatorAlpha", float),
+                ("ThicknessMean_InterpolatedStructure", float),
+                ("ThicknessMedian_InterpolatedStructure", float),
+                ("ThicknessStdDev_InterpolatedStructure", float),
+                ("ThicknessMean_StructuralPoint", float),
+                ("ThicknessMedian_StructuralPoint", float),
+                ("ThicknessStdDev_StructuralPoint", float),
```

Comment:
Is there a way to make this generic so that the user can specify thickness calculators to keep map2loop being modular?